amodevices/hvs_controller/hvs_controller.py

```diff
@@ -0,0 +1,195 @@
+# -*- coding: utf-8 -*-
+"""
+@author: Jack Mango/Berkeley
+
+Device driver for Matsusada J4-5P and KA-10P high voltage supplies.
+J4-5P and KA-10P high voltage supply output voltages can be set using an analog
+control voltage and measured using an analog monitor voltage. Additionally,
+KA-10P high voltage supply current can be measured using a monitor voltage. These
+analog voltages are using a National Instruments 9205 ADC and are set using a 
```

Comment:
"...are set using..."

Reply:
Should I clarify this? Realized this sounds a little repetitive so updated it: how's this read,
```"""This is a device driver designed for the Matsusada J4-5P and KA-10P
high voltage supplies. These supplies allow for the adjustment of output
voltages through analog control voltage inputs, and their outputs can be
monitored via analog monitor voltages. Moreover, the KA-10P high voltage supply
permits the measurement of current using a monitor voltage. The analog voltages
are interfaced with a National Instruments 9205 ADC for measurement and a
National Instruments 9264 DAC for voltage adjustment."""```